### PR TITLE
VAGOVTEAM-8188: Removing nickname field from cms form and view.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See https://github.com/department-of-veterans-affairs/va.gov-team/issues/8188

## Testing done
 - Build cms and then vets-website locally - build success

## Screenshots
N/A

## QA steps
 - As an admin, visit `http://va-gov-cms.lndo.site/admin/structure/types/manage/health_care_region_page/form-display` and `http://va-gov-cms.lndo.site/admin/structure/types/manage/health_care_local_facility/form-display`, visually verifying `field_nickname_for_this_facility` is disabled

### CODE REVIEW

File: config/sync/core.entity_form_display.node.health_care_local_facility.default.yml [hunk 1]
```diff
@@ -61,7 +61,6 @@ third_party_settings:
     group_editorial_workflow:
       children:
         - moderation_state
-        - revision_log
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.node.health_care_local_facility.default.yml:64]: I am not seeing anything in the issue about removing this field.   Was this intentional?

File: config/sync/core.entity_view_display.node.health_care_region_page.default.yml [hunk 2]
```diff
@@ -57,7 +57,7 @@ third_party_settings:
       children:
         - field_intro_text
         - field_media
-        - field_nickname_for_this_facility
+        - field_vamc_system_official_name
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.node.health_care_region_page.default.yml:60]: We are swapping one field for another?

Comment (kevwalsh) [reply config/sync/core.entity_view_display.node.health_care_region_page.default.yml:60]: This makes sense to me, the design decision is to stop using the nickname field. As an aside, the entity view display ("content proofing" screen) for this bundle needs a lot of work, outside the scope of this ticket.

File: config/sync/core.entity_form_display.node.health_care_local_facility.default.yml (hunk 1 above)
Comment (kevwalsh) [reply config/sync/core.entity_form_display.node.health_care_local_facility.default.yml:64]: Seems related to https://github.com/department-of-veterans-affairs/va.gov-cms/pull/1511.

**Comment by swirtSJW** [comment]
I am not sure this PR environment  built correctly
image

**Comment by swirtSJW** [comment]
http://pr1540.ci.cms.va.gov/admin/structure/types/manage/health_care_local_facility/form-display

image

**Comment by swirtSJW** [comment]
http://pr1540.ci.cms.va.gov/admin/structure/types/manage/health_care_region_page/form-display

image

**Comment by swirtSJW** [comment]
Passing now:
image


image


And log revision field is in place.

**Pull request merged**